tox.ini

```diff
@@ -125,7 +144,7 @@ commands =
 
 
 [testenv:manifest]
-basepython = python3.9
+basepython = python3.11
```

Comment:
Why 11 here?

Reply:
3.12 didn't work, simple as that. 🙂 I was planning to come back and investigate, but I wanted to get the build green first.